index.html

```diff
@@ -1,20 +1,3448 @@
+                    </h4>
+                  </div>
+                </div>
+              </aside>
+            </div>
+          </div>
+        </div>
+        <!--.section-picks-->
+
+        <div class="subscribe-block p-1 border-top">
+          <div class="flex-md row">
+            <div class="col-12 col-lg-7 mx-0 px-0">
+              <section class="subscon py-5">
+                <div class="substitle">Subscribe</div>
+                <div class="subsmain p-4">
+                  <div class="container-fluid">
+                    <div class="row">
+                      <div class="col-12 col-md-5 mx-0 px-0">
+                        <div class="imgssubs position-relative">
+                          <div class="backleft overlap">
+                            <a href="#">
+                              <img
+                                alt=""
+                                class=""
+                                src="https://d.newsweek.com/en/full/1710955/cover-january-29-february-05-2021.jpg?w=170&amp;q=95&amp;f=fd74b343707677384d1069d7fa4be2ca"
+                              />
+                            </a>
+                          </div>
+                          <div class="frontright overlap">
+                            <a href="#">
+                              <img
+                                alt=""
+                                class=""
+                                src="https://d.newsweek.com/en/full/1724310/cover-february-12-2021.jpg?w=170&amp;q=95&amp;f=4c020c4b61184a0be65dbbc8e376ab66"
+                              />
+                            </a>
+                          </div>
+                        </div>
+                      </div>
+                      <!--.imgssubs-->
+                      <div
+                        class="txtsubs col-12 col-md-7 text-center pt-2 mx-0 px-0"
+                      >
+                        <h3>
+                          In-depth Stories You Don't See Elsewhere About Topics
+                          You Want to Read!
+                        </h3>
+                        <p>
+                          All Access + Weekly Delivery.
+                          <br />
+                          Choose A Membership That's Perfect for You!
+                        </p>
+                        <button class="btnsubs" type="button">SUBSCRIBE</button>
+                      </div>
+                      <!--.txtsubs-->
+                    </div>
+                  </div>
+                </div>
+              </section>
+              <!--.subscon-->
+            </div>
+            <div class="col-12 col-lg-5 mx-0 px-0">
+              <section class="newscon py-5 flex-md-column">
+                <div class="substitle">Newsletter</div>
+                <div class="newsmain p-4">
+                  <div class="imgnews mx-0 px-0">
+                    <div
+                      class="actimgnews"
+                      style="
```

Comment:
- [ ] All styles should be in the CSS stylesheet. The only ones allowed are the ones that are part of the SVG images. 
- [ ] [OPTIONAL] The SVG html code can be saved into a `.svg` file and the called inside the HTML, this will decrease the amount of HTML that only creates noise in the `index.html` file.